backend/api/src/system/package/mod.rs

```diff
@@ -927,330 +928,283 @@ mod tests {
         Ok(())
     }
 
-    #[cfg(test)]
-    mod tests {
```

Comment:
@bobmcwhirter This looks like more changes than it is. I just moved all the tests beneath a single `tests` module.